.github/workflows/ci.yml

```diff
@@ -645,6 +645,14 @@ jobs:
           breeze release-management publish-docs
           --override-versioned
           ${{ needs.build-info.outputs.docs-filter-list-as-string }}
+      - name: "Generate back-reference for providers"
+        run: breeze release-management add-back-references all-providers
+      - name: "Generate back-reference for apache-airflow"
+        run: breeze release-management add-back-references apache-airflow
+      - name: "Generate back-reference for docker-stack"
+        run: breeze release-management add-back-references docker-stack
+      - name: "Generate back-reference for helm-chart"
+        run: breeze release-management add-back-references helm-chart
```

Comment:
small nit: for each command's title, can we do: `Generate back references for providers`, `Generate back references for apache-airflow`, and so on..

Reply:
Sure. Let me just see if it succeeds with the current build and I will push a fixup